frontend/app/src/main/java/com/example/speechbuddy/HomeActivity.kt

```diff
@@ -1,29 +1,51 @@
 package com.example.speechbuddy
 
+import android.content.Context
 import android.content.Intent
 import android.os.Bundle
+import android.util.Log
```

Comment:
log하고 context는 안 써서 제거해도 될 것 같아요
```suggestion
import android.content.Intent
import android.os.Bundle
```